### DISCUSSION TITLE
using smart connections with multiple devices accessing a cloud shared Obsidian vault

### DISCUSSION THREAD

**Discussion by avataraustin** [brianpetro/obsidian-smart-connections#505] [General]
I sometimes access my Obsidian vault stored on iCloud as a cloud shared vault.  I opened Obsidian on my other laptop and it went through a somewhat lengthy sync process but it updated and was working perfectly.  I then went on vacation a week later and when I opened Obsidian on that laptop it went through the process of creating new smart connections with all my notes and then it wanted to go through the whole local embedding process all over again even though I had already done this last week and even on this laptop itself.  Is there any way to avoid having to re-index and re-embed all my vault every time? I have only been using 2 different devices but this has happened before.

**Comment by brianpetro** [comment]
@avataraustin it could be related to this bug https://github.com/brianpetro/obsidian-smart-connections/issues/496
